conga-aem-definitions/src/main/roles/aem-dispatcher-ams.yaml

```diff
@@ -0,0 +1,518 @@
+  variants:
+  - aem-publish
+  template: conf.dispatcher.d/vhosts/virtualhosts.any.hbs
+
+# --- utils ---
+
+- file: deploy.sh
+  dir: utils/aem-dispatcher-ams
+  variants:
+  - aem-publish
+  template: utils/aem-dispatcher-ams/deploy.sh.hbs
+
+- file: flush.sh
+  dir: utils/aem-dispatcher-ams
+  variants:
+    - aem-publish
+  template: utils/aem-dispatcher-ams/flush.sh.hbs
+
+config:
+
+  httpd:
+
+    # Server and alias names
+    # On publish instances the server name defaults to the tenant name.
+    serverName:
+    serverAliasNames:
+
+    # Logging configuration for httpd
+    logging:
+      level: warn
+      # Access logging similar to "Combined", but without IP address (privacy protection)
+      accessLogFormat: '"0.0.0.0 %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-agent}i\""'
+
+    ssl:
+      # If set to true all requests are redirected to HTTPS and HSTS is enabled
+      enforce: true
+
+    # Redirecting from "/" (only publish dispatcher)
+    rootRedirect:
+      #url: /en.html
+      httpStatus: 301
+
+    # Configure CORS header to be sent to the client (only on publish dispatcher)
+    corsHeader:
+      enabled: true
+      # Generate them by default only for font-related files
+      locationMatch: "\\.(svg|eot|ttf|woff)"
+      # If set to true always "*" is sent as 'Access-Control-Allow-Origin' header
+      # If set to false only the requesting host is all if it is in the hostWhiteList (or this list is empty), and not in the hostBlackList
+      allowAllHosts: true
+      #hostWhiteList:
+      #- "https://www.host1.com"
+      #- "https://www.host2.com"
+
+    headers:
+      # Enables/Configures the Content-Security-Policy header on publish dispatcher (see https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP)
+      contentSecurityPolicy:
+      # Enables/Configures the Referrer-Policy header on publish dispatcher (see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy)
+      referrerPolicy: "origin-when-cross-origin"
+      # Enables/Configures the X-XSS-Protection header on publish dispatcher (see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection)
+      xssProtection:
+      # Enables/Configures the Cache-Control header on publish dispatcher (see https://developer.mozilla.org/en/docs/Web/HTTP/Headers/Cache-Control)
+      cacheControl:
+        # Cache AEM Clientlibs with long cache key for 1 year
+        - locationMatch: "\\.lc-.*-lc\\.min\\.(css|js)$"
+          directives: "max-age=31536000, public"
+      # Set the default experiation time for text/html responses (enabled by default in AEM Cloud Service webserver)
+      htmlExpirationTimeMin: 5
+
+    # List of rewrite rules to include in the vhost
+    rewriteIncludes:
+      - conf.d/rewrites/rewrite.rules
+
+    # Rewrite rules, mainly used to protect critical urls
+    # These rules are evaluated before the mapping rules
+    rewrites:
+      - comment: Security - deny access to product critical urls
```

Comment:
currently we usually do not generate comments in the generated files, but put all comments (as comment) in the yaml file. not sure if its worth breaking the convention here.

Reply:
I have removed the comment option. In the rewrites section it was not used anyways. I have also updated the `uriAllowPostReqest` to match the format of `httpd.mapping.uriAllowContentAccess`

Reply:
and make sure to fix the typo in `uriAllowPostReqest`

Reply:
Oh, i overlooked this several times. This is now fixed.